{"@t":"2024-06-19T05:54:22.0414894Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.12.2.","TimingId":"e2a4870","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:22.1113777Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"HPPlc","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"C:\\Users\\rakbalai\\Downloads\\HPPlc_PreProd\\PLC\\HPPlc\\","MachineName":"WKWEJ651391633","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:22.3306954Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:22.3336175Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:25.4890466Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"3285a26","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:25.8803045Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":389,"TimingId":"3285a26","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:28.1486973Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:28.1486973Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:28.1519575Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:29.7638147Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"8aaa0e4","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:31.0281123Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":1264,"TimingId":"8aaa0e4","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:31.1314811Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"2a1db2c","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:31.6796004Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":548,"TimingId":"2a1db2c","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:32.7635239Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:33.0152452Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:38.4523604Z","@mt":"Starting '{MigrationName}'...","MigrationName":"Skybrud.Umbraco.Redirects","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:38.4523604Z","@mt":"At {OrigState}","OrigState":"2.1.1","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:38.4538220Z","@mt":"Done (pending scope completion).","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:38.4709628Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:38.4780023Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":16461,"TimingId":"e2a4870","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"e5dae887-6b61-4d29-af0e-a5039c9b07f5"}
{"@t":"2024-06-19T05:54:39.4136769Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":8,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"9cc84848-12b5-43ea-947f-f612e23db7b6"}
{"@t":"2024-06-19T05:54:39.4195858Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":8,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"9cc84848-12b5-43ea-947f-f612e23db7b6"}
{"@t":"2024-06-19T05:54:41.8063667Z","@mt":"The current Document Type {ContentTypeAlias} matches a locally declared controller of type {ControllerName}. Custom Controllers for Umbraco routing must implement '{UmbracoRenderController}' and inherit from '{UmbracoControllerBase}'.","@l":"Warning","ContentTypeAlias":"home","ControllerName":"HPPlc.Controllers.HomeController","UmbracoRenderController":"Umbraco.Web.Mvc.IRenderController","UmbracoControllerBase":"System.Web.Mvc.ControllerBase","SourceContext":"Umbraco.Web.Mvc.RenderRouteHandler","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":7,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN ","HttpRequestNumber":3,"HttpRequestId":"357592a9-ccf6-471e-94ef-a7e2b02ed633"}
{"@t":"2024-06-19T05:54:55.1433181Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username sona.s@hp.com from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"e6c7c78c-0037-44f4-a7a8-7165ece0a424"}
{"@t":"2024-06-19T05:54:55.1443173Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: sona.s@hp.com logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"e6c7c78c-0037-44f4-a7a8-7165ece0a424"}
{"@t":"2024-06-19T05:57:45.2501635Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T05:58:06.5165480Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Error","HealthCheckMessage":"Unable to find <strong>'/configuration/system.web/customErrors/@mode'</strong> in config file <strong>'C:\\Users\\rakbalai\\Downloads\\HPPlc_PreProd\\PLC\\HPPlc\\Web.config'</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Error","HealthCheckMessage":"Debug compilation mode is currently enabled. It is recommended to disable this setting before go live.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Warning","HealthCheckMessage":"[httpsCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL https://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database data integrity check","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All document paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T05:58:06.5175472Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All media paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":28,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T06:02:49.3671798Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":125,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T06:07:53.4857336Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":158,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T06:12:57.5544076Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":193,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T06:14:27.2882095Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: sona.s@hp.com, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":196,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"18ed59f9-36e9-4def-8bab-d0a65d42943a"}
{"@t":"2024-06-19T06:18:01.6284981Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":139,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T06:23:05.7046591Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":186,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T06:28:09.7751631Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":224,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T06:33:13.8437844Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":321,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T07:29:59.2026467Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username sona.s@hp.com from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":316,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"edae7aaf-70e6-4fa5-89db-12178fa19072"}
{"@t":"2024-06-19T07:29:59.2026467Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: sona.s@hp.com logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":316,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"edae7aaf-70e6-4fa5-89db-12178fa19072"}
{"@t":"2024-06-19T07:34:30.7562637Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":269,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T07:39:34.8596428Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":32,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T07:44:38.9256541Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":75,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T07:46:37.1103375Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"Whatsapp Notifications","ContentId":0,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":30,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"fbfa1be9-c82b-4778-b829-94cd04d3ea0b"}
{"@t":"2024-06-19T07:48:18.5864413Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"Whatsapp Notifications","ContentId":39460,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":89,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":8,"HttpRequestId":"c24e6aa2-7728-436d-97f6-836677530f51"}
{"@t":"2024-06-19T07:49:43.1291794Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":20,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T07:54:47.2111850Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":253,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T07:59:51.2850109Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":229,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:04:55.3860000Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":101,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:09:36.3898763Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: sona.s@hp.com, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":297,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":9,"HttpRequestId":"3ad97f29-088d-4df9-b130-88fe744265f8"}
{"@t":"2024-06-19T08:09:59.4751968Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":290,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:15:03.5852408Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":262,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:20:07.6763692Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":248,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:25:11.7555494Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":248,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:30:15.8406538Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":87,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:35:19.9193856Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":53,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:40:23.9973508Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":61,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:45:28.0904226Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":223,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:50:32.1484561Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":29196,"ProcessName":"iisexpress","ThreadId":240,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:54:47.7368248Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.12.2.","TimingId":"c439684","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:47.7598746Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"HPPlc","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"C:\\Users\\rakbalai\\Downloads\\HPPlc_PreProd\\PLC\\HPPlc\\","MachineName":"WKWEJ651391633","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:47.8284126Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:47.8284126Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.Runtime.MainDom","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:49.5846369Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"1fed001","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:49.7403526Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":141,"TimingId":"1fed001","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:50.4569072Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:50.4569072Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:50.4569072Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:51.2111564Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"79ca211","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:51.6116329Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":400,"TimingId":"79ca211","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:51.6377972Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"5aa5584","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:51.8043847Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":160,"TimingId":"5aa5584","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:52.1389687Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:52.2166947Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:53.9965773Z","@mt":"Starting '{MigrationName}'...","MigrationName":"Skybrud.Umbraco.Redirects","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:53.9965773Z","@mt":"At {OrigState}","OrigState":"2.1.1","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:53.9965773Z","@mt":"Done (pending scope completion).","SourceContext":"Umbraco.Core.Migrations.MigrationPlan","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:54.0045782Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"InternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:54.0055782Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"MembersIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:54.0065778Z","@mt":"Forcing index {IndexerName} to be unlocked since it was left in a locked state","IndexerName":"ExternalIndex","SourceContext":"Umbraco.Examine.ExamineExtensions","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:54.0075772Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:54.0165778Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":6282,"TimingId":"c439684","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"73b7adbb-4961-40b7-a5e8-41965fec5645"}
{"@t":"2024-06-19T08:54:54.3792032Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"0e6dda0d-1f49-4cc2-8b8b-2505bab40f56"}
{"@t":"2024-06-19T08:54:54.3882021Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"0e6dda0d-1f49-4cc2-8b8b-2505bab40f56"}
{"@t":"2024-06-19T08:54:54.3892026Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"f49d6d28-6cac-4309-aa7f-eecbd72e27d9"}
{"@t":"2024-06-19T08:54:55.2412718Z","@mt":"The current Document Type {ContentTypeAlias} matches a locally declared controller of type {ControllerName}. Custom Controllers for Umbraco routing must implement '{UmbracoRenderController}' and inherit from '{UmbracoControllerBase}'.","@l":"Warning","ContentTypeAlias":"home","ControllerName":"HPPlc.Controllers.HomeController","UmbracoRenderController":"Umbraco.Web.Mvc.IRenderController","UmbracoControllerBase":"System.Web.Mvc.ControllerBase","SourceContext":"Umbraco.Web.Mvc.RenderRouteHandler","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN ","HttpRequestNumber":2,"HttpRequestId":"0e6dda0d-1f49-4cc2-8b8b-2505bab40f56"}
{"@t":"2024-06-19T08:55:14.8980868Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username sona.s@hp.com from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"73338901-0206-49bc-aa53-975c1599c98f"}
{"@t":"2024-06-19T08:55:14.8980868Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: sona.s@hp.com logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":6,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":4,"HttpRequestId":"73338901-0206-49bc-aa53-975c1599c98f"}
{"@t":"2024-06-19T08:57:59.0964525Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Error","HealthCheckMessage":"Unable to find <strong>'/configuration/system.web/customErrors/@mode'</strong> in config file <strong>'C:\\Users\\rakbalai\\Downloads\\HPPlc_PreProd\\PLC\\HPPlc\\Web.config'</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Error","HealthCheckMessage":"Debug compilation mode is currently enabled. It is recommended to disable this setting before go live.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Warning","HealthCheckMessage":"[httpsCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"WARN "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL https://localhost/umbraco - 'Unable to connect to the remote server'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Database data integrity check","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All document paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T08:58:20.4647271Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Database data integrity check","HealthCheckResult":"Success","HealthCheckMessage":"<p>All media paths are valid</p>\r\n","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":34,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO "}
{"@t":"2024-06-19T09:03:03.1903427Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":47,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T09:08:07.2744939Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":133,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T10:47:31.7605968Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":133,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T10:52:35.8502522Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":76,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T12:38:34.7431295Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":58,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T14:15:18.5713968Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":78,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T14:15:56.5754394Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username sona.s@hp.com from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":62,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"99701c48-e667-4867-b458-8be6ad99c27e"}
{"@t":"2024-06-19T14:15:56.5754394Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: sona.s@hp.com logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":62,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":5,"HttpRequestId":"99701c48-e667-4867-b458-8be6ad99c27e"}
{"@t":"2024-06-19T14:20:23.0269728Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":96,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
{"@t":"2024-06-19T14:20:51.4308833Z","@mt":"Document {ContentName} (id={ContentId}) has been published.","ContentName":"Whatsapp Worksheets Notifications","ContentId":39460,"SourceContext":"Umbraco.Core.Services.Implement.ContentService","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":67,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"4cedd695-b42a-4c96-9948-7bbf214dcea0"}
{"@t":"2024-06-19T14:25:27.1481745Z","@mt":"Keep alive failed (at '{keepAlivePingUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:80\r\n   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- End of inner exception stack trace ---\r\n   at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- End of inner exception stack trace ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext() in D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:line 69","keepAlivePingUrl":"http://localhost/umbraco/api/keepalive/ping","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":28100,"ProcessName":"iisexpress","ThreadId":87,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"WKWEJ651391633","Log4NetLevel":"ERROR"}
